### ISSUE TITLE
Avogadro 1.2.0 crashes on start (core dumped) - libfreetype issue?

### ISSUE THREAD

**Issue by shrx** [cryos/avogadro#848]
```
$ avogadro --version
"Avogadro version:	1.2.0	Git:	
LibAvogadro version:	1.2.0	Git:	" 
Locale:  "en_US" 
Libavogadro translations not found. 
Avogadro: 	1.2.0 (Hash )
LibAvogadro: 	1.2.0 (Hash )
Qt: 		4.8.7
OpenBabel: 	2.4.1
```

```
$ avogadro                  
"Avogadro version:	1.2.0	Git:	
LibAvogadro version:	1.2.0	Git:	" 
Locale:  "en_US" 
Libavogadro translations not found. 
System has OpenGL support. 
About to test OpenGL capabilities. 
OpenGL capabilities found: 
	Double Buffering.
	Direct Rendering.
	Antialiasing.
[1]    24080 segmentation fault (core dumped)  avogadro
```

Let me know if you need more info, but there is nothing else printed when I start it.

**Comment by shrx** [comment]
I got some more info with `gdb`:
```
(gdb) run
Starting program: /usr/local/bin/avogadro 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
"Avogadro version:	1.2.0	Git:	
LibAvogadro version:	1.2.0	Git:	" 
Locale:  "en_US" 
Libavogadro translations not found. 
System has OpenGL support. 
About to test OpenGL capabilities. 
OpenGL capabilities found: 
	Double Buffering.
	Direct Rendering.
	Antialiasing.

Program received signal SIGSEGV, Segmentation fault.
0x00007ffff4f59781 in ?? () from /usr/lib/x86_64-linux-gnu/libfreetype.so.6
```

**Comment by shrx** [comment]
Solved by downgrading libfreetype6 from 2.7.0 to 2.6.3.
